tutorials/misc/plot_ecog.py

```diff
@@ -183,20 +166,118 @@ def animate(i, activity):
                                frames=show_power.shape[1],
                                interval=100, blit=True)
 
+###############################################################################
+# Now let's animate the raw time series data and add events related
+# to seizure onset that are marked in the annotations. This will be done
+# very similarly to the high gamma visualization.
+#
+# As can be seen in the plot below, the seizure originates in the temporal
+# lobe and spreads to new electrodes when electrode label names come up in
+# the title. The seizure eventually becomes generalized and then subsides.
+
+# colormap to view spectral power - RdBu_r works well for directional
+# data such as voltage time series, where positive and negative values
+# matter and there is a true 0 point
+cmap = 'RdBu_r'
+
+# Apply a notch filter to remove 60 Hz line noise and harmonics
+raw_notched = raw.copy().pick_types(
+    meg=False, ecog=True).notch_filter([60, 120])
+
+# Apply a gentle high pass filter to get rid of drift
+raw_notched.filter(l_freq=0.1, h_freq=None)
+
+# Downsample the data to make the animation more reasonable
+sfreq = 50  # Hz
+raw_notched.resample(sfreq)
+ts_data = raw_notched.get_data()
```

Comment:
@larsoner added an initial downsample to 200 Hz, but I see here you're downsampling to 50.  Could we just go straight to 50 up above instead of doing it twice?

Reply:
We wanted to characterize the "gamma band" power, which has frequencies from "30-90" traditionally. If still doesn't make sense to do it here, let us know.

Reply:
ah, right, so up above we need at least 200 Hz sfreq.  And I assume that's still too high for the animation to work smoothly / compute in a reasonable amount of time?  In that case let's just change the code comment above (see other comment)

Reply:
Right, to compute gamma band power we need the higher sampling rate, but the point of downsampling here was because of the animation. I suppose this could happen for the gamma power data too, once it's computed, and that could save space/memory. However, we should not downsample to 50 Hz *before* computing the gamma power.